packages/keyv-mongo/src/index.ts

```diff
@@ -0,0 +1,112 @@
+import EventEmitter from 'events';
+import { MongoClient, Collection } from 'mongodb';
+import { KeyvStore, KeyvStoreObject } from 'keyv';
+
+export interface KeyvMongoOptions {
+	collection: string;
+	url: string;
+	uri?: string;
+}
+
+interface Document<TVal> {
+	key: string;
+	value: KeyvStoreObject<TVal>;
+	expiresAt: Date | null;
+}
+
+export default class KeyvMongo<TVal> extends EventEmitter implements KeyvStore<TVal, KeyvStoreObject<TVal>> {
```

Comment:
`implements` forces the class to correctly implement the interface. in this case, it forces `get`, `set`, etc. to abide by the standard api for stores (`KeyvStore`)